components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -244,26 +246,70 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
     }, []);
 
     const accData: AccordionItemType[] = Object.entries(data).map(([accordionKey, accordionData]) => {
-        const items = accordionData.items.map((item) => (
-            <AccordionItem
-                key={`${accordionKey}-item_${item.id}`}
-                iconColor={getItemColor(item.status)}
-            >
-                <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
-                <p>{item.description}</p>
-            </AccordionItem>
-        ));
+        const chipsInfo: { [key in ChipsInfoKey]: number } = {
+            info: 0,
+            warning: 0,
+            error: 0,
+            none: 0,
+        };
+        const items: React.ReactNode[] = [];
+        accordionData.items.forEach((item) => {
+            items.push((
+                <AccordionItem
+                    key={`${accordionKey}-item_${item.id}`}
+                    iconColor={getItemColor(item.status)}
+                >
+                    <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
```

Comment:
nit: this line is a bit long ... could we break that up into multiple lines please?

Reply:
if you are ok with the other proposals we should change the first conditional class to `'icon-check-circle-outline': !item.status,`

Reply:
and are we missing the suggestion/info here?